app/src/main/java/com/globant/scriptsapadea/ui/activities/MainActivity.java

```diff
@@ -81,4 +98,20 @@ public void clickEdit(View view) {
     public void clickReadScript(View view) {
         Log.i("INFO", "clickReadScript");
     }
+
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+        bus.post(new ActivityResultEvent(requestCode, resultCode, data));
+    }
+
+    // TODO: Use Otto to create this event
+    @Override
+    public void onChangeFragment(Fragment fragment, boolean addInStack) {
+        if (addInStack)
```

Comment:
I added addInStack because if you want to push another fragment into stack you should notify to callback. The navigator manages this feature but if you don't want to push fragment in stack you need to use the noPush event so I think it's better to nootify to callback if you want to push fragment in stack. What do you think Nicolás?